config/peerpods/podvm/lib.sh

```diff
@@ -61,13 +62,15 @@ function install_rpm_packages() {
 #"packer=https://releases.hashicorp.com/packer/1.9.4/packer_1.9.4_linux_amd64.zip"
 #"kubectl=https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.14.9/openshift-client-linux.tar.gz"
 #"yq=https://github.com/mikefarah/yq/releases/download/v4.35.2/yq_linux_amd64.tar.gz"
+#"umoci=https://github.com/opencontainers/umoci/releases/download/v0.4.7/umoci.amd64"
```

Comment:
we'll need a way to configure it once multiarch is supported

Reply:
Yeah, all the binaries are x86_64 for now and needs to be switched to multi-arch. I'll add a TBD